client/my-sites/plugins/marketplace/components/marketplace-shopping-cart/index.tsx

```diff
@@ -0,0 +1,202 @@
+import styled from '@emotion/styled';
+import { translate } from 'i18n-calypso';
+import { keyframes } from '@emotion/core';
+
+/**
+ * Internal dependencies
+ */
+import { LineItem } from 'calypso/my-sites/checkout/composite-checkout/components/wp-order-review-line-items';
+import { MarketplaceThemeType } from 'calypso/my-sites/plugins/marketplace';
+import { MobileHiddenHorizontalRule } from 'calypso/my-sites/plugins/marketplace/components';
+
+interface PropsForMarketplaceShoppingCart {
+	onCheckout: () => void;
+	selectedDomainProductUUID: string;
+	isExpandedBasketView: boolean;
+	toggleExpandedBasketView: () => void;
+}
+
+const ShoppingCart = styled.div< { theme?: MarketplaceThemeType } >`
+	width: 100%;
+	background-color: var( --studio-gray-0 );
+	padding: 15px 25px;
+	max-width: 611px;
+	display: inline-table;
+	@media ( ${ ( { theme } ) => theme.breakpoints.tabletDown } ) {
+		overflow-y: scroll;
+		-ms-overflow-style: none;
+		scrollbar-width: none;
+		&::-webkit-scrollbar {
+			display: none;
+		}
+		padding: 0 35px;
+	}
+`;
+
+const ShoppingCartTitle = styled.div`
+	font-size: 1.25em;
+	display: flex;
+	justify-content: space-between;
+`;
+
+const MobileTotal = styled.div< { theme?: MarketplaceThemeType } >`
+	display: none;
+	@media ( ${ ( { theme } ) => theme.breakpoints.tabletDown } ) {
+		display: block;
+	}
+`;
+
+const ShoppingCartTotal = styled.div< { theme?: MarketplaceThemeType } >`
+	display: flex;
+	justify-content: space-between;
+	width: 100%;
+	margin-bottom: 20px;
+	div {
+		font-size: 1.15rem;
+		font-weight: ${ ( { theme } ) => theme.weights.bold };
+		font-weight: 600;
+	}
+
+	@media ( ${ ( { theme } ) => theme.breakpoints.tabletDown } ) {
+		display: none;
+	}
+`;
+
+const FullWidthButton = styled( Button )`
+	justify-content: center;
+	width: 100%;
+	margin-bottom: 15px;
+`;
+
+const MarketPlaceBasketItem = styled( LineItem )`
+	div,
+	span {
+		font-size: 0.875rem;
+	}
+	padding: 4px 0;
+`;
+
+const MobileToggleExpandedBasket = styled.a< { theme?: MarketplaceThemeType } >`
+	display: none;
+	background-color: transparent;
+	border-color: transparent;
+	color: var( --color-accent-60 );
+	margin-bottom: 20px;
+	box-shadow: none;
+	cursor: pointer;
+	&:hover {
+		color: var( --color-accent-60 );
+	}
+
+	@media ( ${ ( { theme } ) => theme.breakpoints.tabletDown } ) {
+		display: block;
+	}
+`;
+
+const pulse = keyframes`
+  0% {
+    opacity: 1;
+  }
+
+  70% {
+  	opacity: 0.5;
+  }
+
+  100% {
+    opacity: 1;
+  }
+`;
+
+const SideBarLoadingCopy = styled.p`
+	font-size: 14px;
+	height: 31px;
+	width: 100%;
+	content: '';
+	background: var( --studio-gray-5 );
+	margin: 15px 0 0 0;
+	padding: 0;
+	animation: ${ pulse } 2s ease-in-out infinite;
+`;
+
+const StyledBasketItemContainer = styled( BasketItemContainer )`
+	margin: 8px 0;
+	min-height: 30px;
+`;
+
+function BasketItemContainer( { products, isLoading, isExpandedBasketView } ) {
+	if ( isLoading ) {
+		return (
+			<div>
```

Comment:
Do we need a `div` or we could use a `React.Fragment` / `<></>` instead?

Reply:
There are some styles that depend on a block container element
![image](https://user-images.githubusercontent.com/3422709/117004912-4429da00-ad04-11eb-8c52-23fd67a1dd7f.png)